### PR TITLE
[FHB-54] Days available filter

### PR DESCRIPTION
**Pull request by Top-Cat**

### CODE REVIEW

File: src/ui/connect-ui/src/FamilyHubs.Referral.Web/Pages/ProfessionalReferral/LocalOfferResults.cshtml.cs [hunk 1]
```diff
@@ -138,10 +142,9 @@ public LocalOfferResultsModel(
     }
 
     public async Task<IActionResult> OnGetAsync(
-        string postcode, string? searchText, string? searchAge,
-        string? selectedLanguage, string? subcategorySelection,
-        string? costSelection, string? serviceDeliverySelection,
-        int? pageNum, bool forChildrenAndYoungPeople, Guid? correlationId
+        string postcode, string? searchText, string? searchAge, string? selectedLanguage,
```
Comment (stevesatdfe) [inline src/ui/connect-ui/src/FamilyHubs.Referral.Web/Pages/ProfessionalReferral/LocalOfferResults.cshtml.cs:145]: Things like this feel like an object should be passed in. I think I will suggest a code walkthrough with the team and review these things. No action.

**Review by stevesatdfe — CHANGES_REQUESTED** [review]
Nothing major.

File: src/service/service-directory-api/tests/FamilyHubs.ServiceDirectory.Api.FunctionalTests/WhenUsingServiceApiUnitTests.cs [hunk 2]
```diff
@@ -375,4 +376,33 @@ public async Task ThenTheServicesWithOutFamilyHubsAreRetrieved()
...
+            Method = HttpMethod.Get,
+            RequestUri = new Uri(Client.BaseAddress + $"api/services-simple{url}")
+        };
+
+        using var response = await Client.SendAsync(request);
+
+        var responseContent = await response.Content.ReadAsStringAsync();
+
+        if (!response.IsSuccessStatusCode)
+            ArgumentException.ThrowIfNullOrEmpty(responseContent);
```
[the remark this answers is not in the document]
Comment (Top-Cat) [reply src/service/service-directory-api/tests/FamilyHubs.ServiceDirectory.Api.FunctionalTests/WhenUsingServiceApiUnitTests.cs:400]: Copy pasted but you're right. Might fix the rest of them.

File: src/ui/connect-ui/src/FamilyHubs.Referral.Web/Pages/ProfessionalReferral/LocalOfferResults.cshtml.cs (hunk 1 above)
Comment (Top-Cat) [reply src/ui/connect-ui/src/FamilyHubs.Referral.Web/Pages/ProfessionalReferral/LocalOfferResults.cshtml.cs:144]: I do agree, some of these have a lot of arguments

File: src/ui/connect-ui/src/FamilyHubs.Referral.Web/Pages/ProfessionalReferral/LocalOfferResults.cshtml [hunk 3]
```diff
@@ -254,6 +273,33 @@ else
             </fieldset>
         </div>
 
+        <div class="govuk-form-group fh-filter-group">
+            <fieldset class="govuk-fieldset">
+                <legend class="govuk-fieldset__legend govuk-fieldset__legend--s">
+                    Days service is available
+                </legend>
+                <div class="govuk-checkboxes govuk-checkboxes--small" data-module="govuk-checkboxes">
+                    @for (var day = 0; day < 7; day++)
```
[the remark this answers is not in the document]
Comment (Top-Cat) [reply src/ui/connect-ui/src/FamilyHubs.Referral.Web/Pages/ProfessionalReferral/LocalOfferResults.cshtml]: Swapped to use Enum.GetValues like the other PR

**Review by stevesatdfe — APPROVED** [review]

**Pull request merged**